README.md

```diff
@@ -944,7 +1083,7 @@ cache:
     - $HOME/.cache/trivy
 ```
 
-Example: https://travis-ci.org/knqyf263/trivy-ci-test  
+Example: https://travis-ci.org/knqyf263/trivy-ci-test\
```

Comment:
`\` does not seem to work.
![image](https://user-images.githubusercontent.com/2253692/59027035-d57f4000-8892-11e9-9114-ca2bc9374ddb.png)